Components/Login.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import { Text, View, TextInput } from "react-native";
+import { Button } from "native-base";
+import api from "../Utils/apiUtils";
+import styles from "./Styling/formStyling";
+import HeaderBar from "./HeaderBar";
+import { ScrollView } from "react-native-gesture-handler";
+
+class Login extends Component {
+  state = {
+    firstName: null,
+    user_id: "Enter your Username",
+    password: "Enter your password"
+  };
+
+  onPressButton = () => {
+    const { user_id } = this.state;
+    api
+      .getUser(user_id)
+      .then(user => {
+        this.setState({ user_id: user.id, firstName: user.first_name });
+      })
+      .then(() => {
+        this.props.screenProps.setUser(this.state.user_id);
+      });
+  };
+
+  render() {
+    return (
+      <View>
+        <HeaderBar />
+        <ScrollView>
+          <Text style={styles.formTitle}>Login</Text>
+          <Text style={styles.formText}>Username</Text>
+          <TextInput
```

Comment:
https://facebook.github.io/react-native/docs/textinput#placeholder

perhaps use this prop for the "enter username here?" 
```suggestion
          <TextInput placeholder="Enter your Username"
```